### ISSUE TITLE
Hover to inspect variables not working on VSCode 1.8.0

### ISSUE THREAD

**Issue by G2Jose** [Microsoft/vscode-chrome-debug#309]
- VSCode Version: 1.8.0
- OS Version: MacOS Sierra

Steps to Reproduce:

1. Install the _Debugger for Chrome_ addon
2. Set up launch.json

```
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch Chrome against localhost, with sourcemaps",
            "type": "chrome",
            "request": "launch",
            "url": "http://localhost:8080",
            "sourceMaps": true,
            "webRoot": "${workspaceRoot}"
        },
        {
            "name": "Attach to Chrome, with sourcemaps",
            "type": "chrome",
            "request": "attach",
            "port": 9222,
            "sourceMaps": true,
            "webRoot": "${workspaceRoot}"
        }
    ]
}

```
3. Run application in debug mode, and set breakpoints
4. Execution pauses at breakpoints as expected, but you can no longer inspect variables by hovering over them (this used to work with version 1.7)

(Also filed an issue on https://github.com/Microsoft/vscode/issues/17320)

**Comment by roblourens** [comment]
Are you using the latest version of the Chrome extension? There was a change to this, but it should have fixed it, not broken it :)

**Comment by roblourens** [comment]
If you updated vscode but not the chrome extension, then I wouldn't be surprised that it's broken.

**Comment by G2Jose** [comment]
@roblourens I'm using the latest version of chrome debug (v2.4.0).

**Comment by roblourens** [comment]
Can I see the code where hover doesn't work? If possible can I see the entire project?

Is there anywhere that it does work? Maybe close to the top of the file (where a sourcemapping error is less likely to be off)?

**Comment by G2Jose** [comment]
Unfortunately I can't share code from the original project (NDA). I did however create another skeleton project to demonstrate the error (https://github.com/G2Jose/vscode-chrome-debug-temp). For this project, I seem to be having a sourcemap issue, and the breakpoint never even gets verified. 

![Screen Recording](http://i.imgur.com/KBQCiJ4.gif)

**Comment by roblourens** [comment]
I see the problem...

**Comment by roblourens** [comment]
Fixed in 2.4.1, thanks for the reports
